### DISCUSSION TITLE
Create a g0v4git homebrew formula

### DISCUSSION THREAD

**Discussion by patcon** [gov4git/gov4git#194] [Ideas]
This would make it slightly simpler to install and manage the installation :)
https://docs.brew.sh/Formula-Cookbook#an-introduction

Is this in scope, if someone were to submit a PR?

**Comment by petar** [comment]
Thanks for the suggestion. Definitely in scope. This could be useful for power users who want access to a broader set of functionalities than the desktop app covers, as well as folks who don't use GitHub as a git provider. 

If you were to undertake this, please, make sure to include accompanying automation for publishing to brew on release.
